acceptance_tests/utilities/pubsub_helper.py

```diff
@@ -46,20 +46,13 @@ def purge_aims_new_address_subscription():
         subscriber.seek(aims_subscription_path, time=timestamp)
     except MethodNotImplemented as e:
```

Comment:
flake8 complains here as the local variable e isn't being used anymore
```suggestion
    except MethodNotImplemented:
```